API.md

```diff
@@ -1,19 +1,349 @@
-# Party Service API
+###Party Service API
 
-### The implemented API for this serivce can be found at /swagger/party/party-api.yaml
+######*Acceptance tests can be run to populate a local version with test data.*
+<hr>
```

Comment:
what's this doing in here?

in markdown we use three hyphens (i.e. `---`) to indicate a _horizontal rule_ where we believe we need a section separator.